src/scenes/CollectionGalleryPage/CollectionGalleryHeader.tsx

```diff
@@ -223,12 +237,8 @@ const StyledCollectionName = styled.div`
 
 const StyledSeparator = styled.div`
   margin: 0 10px;
-  display: none;
+  display: block;
   color: ${colors.metal};
-
-  @media only screen and ${breakpoints.mobileLarge} {
-    display: block;
-  }
 `;
```

Comment:
The new Figma designs have a `/` in the header text on all screen sizes (e.g. `connorr / My Collection`), so we remove the dynamic `display: ` here